src/DotLiquid/Util/EnumerableExtensionMethods.cs

```diff
@@ -33,5 +33,26 @@ public static void EachWithIndex(this IEnumerable<object> array, Action<object,
                 ++index;
             }
         }
+
+        /// <summary>
+        /// Determines whether a sequence contains any elements.
+        /// </summary>
+        /// <param name="source">The IEnumerable to check for emptiness.</param>
+        /// <returns>true if the source sequence contains any elements; otherwise, false.</returns>

```

Comment:
```suggestion
        /// <returns><see langword="true"/> if the source sequence contains any elements; otherwise, <see langword="false"/>.</returns>
```